pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -244,6 +265,22 @@ func (m *Multicluster) DeleteMemberCluster(clusterID string) error {
 	return nil
 }
 
+func createConfigStore(client kubelib.Client, revision string, opts Options) (model.ConfigStoreCache, error) {
+	log.Infof("Creating WorkloadEntry only config store for %s", opts.ClusterID)
+	workloadEntriesSchemas := collection.NewSchemasBuilder().
+		MustAdd(collections.IstioNetworkingV1Alpha3Workloadentries).
+		Build()
+	configController, err := crdclient.NewForSchemas(client, revision, opts.DomainSuffix, workloadEntriesSchemas)
+	if err != nil {
+		return nil, err
+	}
+	aggregateStore, err := configaggregate.MakeCache([]model.ConfigStoreCache{configController})
```

Comment:
aggregate store of single item is equivalent to configController I would think? is this needed?

Reply:
To clarify, I am asking if `return configController, nil` could be done here instead